content/docs/intro/concepts/programming-model.md

```diff
@@ -140,6 +157,11 @@ db = Database("db", opts=ResourceOptions(additional_secret_outputs=["password"])
 // See https://github.com/pulumi/pulumi/issues/1614.
 ```
 
+```csharp
+// Ensure the password generated for the database is marked as a secret
+var db = new Database("new-name-for-db", new DatabaseArgs(), new CustomResourceOptions { AdditionalSecretOutputs = { "password" } });
```

Comment:
Nit, word wrap? This probably won't fit horizontally on one line nicely on the page.

Reply:
this matches the exmaples above.